UI/keyed-list-widget.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <QString>
+#include <QWidget>
+#include <QList>
+#include <QListWidget>
+#include <QListWidgetItem>
+#include <QAbstractItemView>
+
+#define NONE_SELECTED -1
+
+class KeyedListWidget : public QListWidget {
```

Comment:
Is this class still needed?

Reply:
Good point. I think we can safely remove it now that we don't have to keep track of multiple services in the settings page qt widget. I'll clean things up when I merge the dev branch.